P5/Source/Specs/citeData.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="header" ident="citeData">
+  <gloss versionDate="2020-02-03" xml:lang="en">citation data</gloss>
+  <desc versionDate="2008-01-13" xml:lang="en">specifies how information may be extracted from citation structures.</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="att.citeStructurePart"/>
+  </classes>
+  <content>
+    <empty/>        
+  </content>
+  <attList>
+    <attDef ident="property" usage="req">
```

Comment:
`@property` must be required, because otherwise `<citeData>` won't be able to say what kind of data it represents. But it shouldn't be required to be an RDF property. It might map to a category in a taxonomy or a SKOS Concept, etc.